prow/knative/config.yaml

```diff
@@ -110,7 +110,7 @@ deck:
     - lens:
         name: junit
       required_files:
-      - artifacts/junit.*\.xml
+      - artifacts(/.*/|/)junit.*\.xml
```

Comment:
Can we have a comment here explaining the reason?

Reply:
I've included a brief comment on top